.circleci/config.yml

```diff
@@ -321,9 +332,11 @@ workflows:
           <<: *only_master
           name: build-docker-image-master
           staging: 'true'
+          uat: 'true'
```

Comment:
Do we want to build UAT on every merge to master too? I think we were only linking it to production/pre-production deploys for now?